migrations/1625095491619-CreateBlockedAddressesTable.ts

```diff
@@ -0,0 +1,40 @@
+import { MigrationInterface, QueryRunner, Table, TableCheck, TableIndex } from 'typeorm';
+
+const blockedAddressesTable = new Table({
+    name: 'blocked_addresses',
+    columns: [
+        { name: 'address', type: 'varchar', isPrimary: true },
+        { name: 'created_at', type: 'timestamptz', default: 'NOW()' },
+        { name: 'parent', type: 'varchar', isNullable: true },
+        { name: 'last_seen_nonce', type: 'bigint', isNullable: true },
+        { name: 'ignore', type: 'boolean', default: false },
+    ],
+});
+const createdAtIndex = new TableIndex({ name: `blocked_addresses_created_at_index`, columnNames: ['created_at'] });
+const ignoreIndex = new TableIndex({ name: `blocked_addresses_ignore_index`, columnNames: ['ignore'] });
+
+export class CreateBlockedAddressesTable1625095491619 implements MigrationInterface {
+    public async up(queryRunner: QueryRunner): Promise<void> {
+        const driver = queryRunner.connection.driver;
+        const addressLowerCaseConstraint = new TableCheck({
```

Comment:
nice 👍 

could be worth adding a test for this

Reply:
should at least check locally after running migrations @phil-ociraptor

Reply:
ah yeah, an automated test would be good

And the local version looks good 😎 

```
api=# insert into blocked_addresses (address) values ('HARRY');
ERROR:  new row for relation "blocked_addresses" violates check constraint "CHK_39e5b70f27376dfac15dd3daf3"
DETAIL:  Failing row contains (HARRY, 2021-07-01 23:33:34.704671+00, null, null, f).
api=# insert into blocked_addresses (address) values ('harry');
INSERT 0 1
```